src/streamlink/logger.py

```diff
@@ -46,7 +46,7 @@ def getMessage(self):
         return msg
 
 
-class StreamlinkLogger(logging.getLoggerClass()):
+class StreamlinkLogger(logging.getLoggerClass(), object):
```

Comment:
Not of the effect of this, it seems fine - but is it really needed?

Reply:
This is required for python 2.6, else I was getting TypeError. Such as this: https://stackoverflow.com/questions/1713038/super-fails-with-error-typeerror-argument-1-must-be-type-not-classobj-when

Reply:
Fair enough :)